### PR TITLE
Flux Sensu package

### LINKED ISSUE #1217: Add Flux Sensu package
PR: https://github.com/influxdata/flux/pull/3000

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1217 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@sranka Feel free to take a look at this PR.

File: content/v2.0/reference/flux/stdlib/contrib/sensu/endpoint.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  using data from table rows.
+menu:
+  v2_0_ref:
+    name: sensu.endpoint
+    parent: Sensu
+weight: 202
+related:
+  - https://docs.sensu.io/sensu-go/latest/api/events/, Sensu Events API
+  - https://docs.sensu.io/sensu-go/latest/api/apikeys/, Sensu APIKeys API
+  - https://docs.sensu.io/sensu-go/latest/reference/handlers/, Sensu Handlers
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/sensu/endpoint.md:15]: handlers?

File: content/v2.0/reference/flux/stdlib/contrib/sensu/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Import the `contrib/sranka/sensu` package:
+
+```js
+import "contrib/sranka/sensu"
+```
+
+{{< children type="functions" show="pages" >}}
+
+## Sensu API Key authentication
+The Flux Sensu package only supports [Sensu API Key authentication](https://docs.sensu.io/sensu-go/latest/api/#authenticate-with-an-api-key).
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/sensu/_index.md:27]: API key?

File: content/v2.0/reference/flux/stdlib/contrib/sensu/endpoint.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+### namespace
+[Sensu namespace](https://docs.sensu.io/sensu-go/latest/reference/rbac/).
+Default is `default`.
+
+_**Data type:** String_
+
+### entityName
+Event source.
+Use alpha-numeric characters, underscores (`_`), periods (`.`), and hyphens (`-`).
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/sensu/endpoint.md:65]: alphanumeric

File: content/v2.0/reference/flux/stdlib/contrib/sensu/event.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  [Sensu Events API](https://docs.sensu.io/sensu-go/latest/api/events/#create-a-new-event).
+menu:
+  v2_0_ref:
+    name: sensu.event
+    parent: Sensu
+weight: 202
+related:
+  - https://docs.sensu.io/sensu-go/latest/api/events/, Sensu Events API
+  - https://docs.sensu.io/sensu-go/latest/api/apikeys/, Sensu APIKeys API
+  - https://docs.sensu.io/sensu-go/latest/reference/handlers/, Sensu Handlers
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/sensu/event.md:14]: handlers?

**Pull request merged**